apps/auth/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import include
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No name 'include' in module 'django.urls'](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=5459956)